tephra-hbase-compat-0.98/src/main/java/co/cask/tephra/hbase98/coprocessor/TransactionProcessor.java

```diff
@@ -151,7 +151,7 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     if (tx != null) {
       projectFamilyDeletes(get);
       get.setMaxVersions();
-      get.setTimeRange(TxUtils.getOldestVisibleTimestamp(ttlByFamily, tx), TxUtils.getMaxVisibleTimestamp(tx));
+      get.setTimeRange(0, TxUtils.getMaxVisibleTimestamp(tx));
```

Comment:
Good point, I'll add `(current time - TTL)` in milliseconds as the lower bound for now. But for a purely transactional table this is still equivalent to having no lower bound. We should see if we can do better.